### PR TITLE
Feature/manifest v3 migration

### PR DESCRIPTION
**Pull request by jlf0dev**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Add an initial manifest v3 build for browser
## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

```
- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)
```

### CODE REVIEW

File: apps/browser/package.json [hunk 1]
```diff
@@ -4,9 +4,11 @@
   "scripts": {
     "build": "webpack",
     "build:watch": "webpack --watch",
+    "build:watch:MV3": "MANIFEST_VERSION=3 webpack --watch",
```
Comment (justindbaur) [inline apps/browser/package.json:7]: Will this set the environment variable? For `NODE_ENV` we use `cross-env` to do it. Should we do that for this as well?

**Review by justindbaur — COMMENTED** [review]
One small question

File: apps/browser/package.json (hunk 1 above)
Comment (jlf0dev) [reply apps/browser/package.json]: Good catch, took it out while debugging and forgot to add it back in

File: apps/browser/src/manifest.json.v3 [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  "permissions": [
+    "tabs",
+    "contextMenus",
+    "storage",
+    "unlimitedStorage",
+    "clipboardRead",
+    "clipboardWrite",
+    "idle",
+    "webRequest",
+    "declarativeNetRequest"
```
[the remark this answers is not in the document]
Comment (jlf0dev) [reply apps/browser/src/manifest.json.v3:69]: I'll leave it in for now, once we know for sure we can pull it out

**Review by justindbaur — APPROVED** [review]
Looks great.

**Review by MGibson1 — APPROVED** [review]

**Pull request merged**